fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -1683,7 +1732,7 @@ struct TenantManagementWorkload : TestWorkload {
 			if (operation == 0) {
 				wait(createTenant(self));
 			} else if (operation == 1) {
-				wait(deleteTenant(self));
+				wait(deleteTenant(self, deterministicRandom()->coinflip()));
```

Comment:
Nit: rather than pass this as an argument, you could move it into the function itself. These functions have a bunch of parameter generation at the top of them where this would fit naturally.